src/main/kotlin/graphics/scenery/volumes/BufferedVolume.kt

```diff
@@ -219,6 +227,13 @@ class BufferedVolume(val ds: VolumeDataSource.RAISource<*>, options: VolumeViewe
         }
     }
 
+    private fun NumericType<*>.maxValue(): Float = when(this) {
```

Comment:
`Volume` has a `range` property now. The max from that should be used instead of this function, which should be removed.

Reply:
On which variable should the range method be called? Also, there's only a `toRange` method in Volume. Or should the transferRangeMax variable be completely replaced with the result of this new method?